### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a section for onenote

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9937

File: microsoft-365/lti/index.md [hunk 1]
```diff
@@ -93,3 +94,9 @@ For configuration steps, see:
 - [Microsoft Teams Classes with Moodle](teams-classes-meetings-with-moodle.md).
 - [Microsoft Teams Classes with Open LMS](open-lms-teams-classes-and-meetings.md).
 - [Microsoft Teams Classes with Desire2Learn Brightspace](teams-classes-meetings-with-brightspace.md).
+
+### OneNote Class Notebook LTI
+
+
+OneNote Class Notebook LTI can work with your Learning Management System.Use OneNote Class Notebook with your LMS to create a shared notebook and link it to your course. Students enrolled in your LMS course can access the notebook automatically without you having to add their names. For configuation steps, See [Microsoft OneNote Class Notebook LTI](https://www.onenote.com/lti/integratelti)
```
Comment (JohanFreelancer9) [inline microsoft-365/lti/index.md]:
```suggestion
OneNote Class Notebook LTI can be used with your Learning Management System (LMS) to create a shared notebook and link it to your course. Students enrolled in your LMS course can access the notebook automatically without you having to add their names. For configuration steps, see [Microsoft OneNote Class Notebook LTI](https://www.onenote.com/lti/integratelti).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@DaniEASmith Please merge this PR. Thanks

**Pull request merged**